Library/Homebrew/test_runner_formula.rb

```diff
@@ -90,24 +90,24 @@ def dependents(platform:, arch:, macos_version:)
     cache_key = :"#{platform}_#{arch}_#{macos_version}"
 
     @dependent_hash[cache_key] ||= begin
-      all = eval_all || Homebrew::EnvConfig.eval_all?
```

Comment:
I assume that `Homebrew::EnvConfig.eval_all?` isn't needed here anymore because we already pass it in manually somewhere else?

Reply:
Yes, all callers already pass this explicitly.